app/routes/admin/sessions/list.js

```diff
@@ -0,0 +1,108 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    switch (this.get('params.sessions_state')) {
+      case 'pending':
+        return this.i18n.t('Pending');
```

Comment:
But one thing I want to clear that in earlier comment @niranjan94 mentioned that dynamic strings won't be marked for translation properly. So I am not sure will this be `this.i18n.t(this.get('params.sessions_state'))` translated properly?
@niranjan94 Please clarify this. Thanks.